device/ios/Plugin/DevicePlugin.swift

```diff
@@ -5,6 +5,11 @@ import Capacitor
 public class DevicePlugin: CAPPlugin {
     private let implementation = Device()
 
+    @objc func getId(_ call: CAPPluginCall) {
+        call.resolve([
+            "uuid": UIDevice.current.identifierForVendor!.uuidString
```

Comment:
Although the previous implementation forced unwrapped the optional, we should avoid doing so where possible. [The documentation](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) is unclear on when the method might return nil; it only provides one example scenario where it can happen. But adding a fallback is easy insurance against a crash.

```suggestion
            "uuid": UIDevice.current.identifierForVendor?.uuidString ?? ""
```

Reply:
I try to not change code when moving it from one place to another.
Anyway, I've made it to reject if the identifier is not available, I think that's better than returning an empty string.